crds/operators.coreos.com_clusterserviceversions.yaml

```diff
@@ -8704,6 +8704,10 @@ spec:
                     containerPort:
                       type: integer
                       format: int32
+                    conversionCrds:
```

Comment:
this should be `conversionCRDs` according to the naming conventions (see section on acronyms: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions)